tests/unit/app/models/test_models.py

```diff
@@ -1,38 +1,92 @@
 """Unit tests for the API models."""
 
-from ols.app.models.models import FeedbackRequest, LLMRequest
-from ols.app.utils import Utils
-
-
-def test_feedback_request():
-    """Test the FeedbackRequest model."""
-    conversation_id = Utils.get_suid()
-
-    feedback_request = FeedbackRequest(
-        conversation_id=conversation_id,
-        feedback_object='{"rating": 5, "comment": "Great service!"}',
-    )
-    assert feedback_request.conversation_id == conversation_id
-    assert (
-        feedback_request.feedback_object == '{"rating": 5, "comment": "Great service!"}'
-    )
-
-
-def test_llm_request():
-    """Test the LLMRequest model."""
-    llm_request = LLMRequest(query="Tell me about Kubernetes")
-    assert llm_request.query == "Tell me about Kubernetes"
-    assert llm_request.conversation_id is None
-    assert llm_request.response is None
-
-    llm_request = LLMRequest(
-        query="Tell me about Kubernetes",
-        conversation_id="abc",
-        response="Kubernetes is a portable, extensible, open source platform ...",
-    )
-    assert llm_request.query == "Tell me about Kubernetes"
-    assert llm_request.conversation_id == "abc"
-    assert (
-        llm_request.response
-        == "Kubernetes is a portable, extensible, open source platform ..."
-    )
+from ols.app.models.models import (
+    FeedbackRequest,
+    FeedbackResponse,
+    HealthResponse,
+    LLMRequest,
+    LLMResponse,
+)
+
+
+class TestLLM:
```

Comment:
what's the purpose of using classes there? Is not it bad behavior?

Reply:
Maybe you meant "bad practice"? I'm not a dog. :P
And I wouldn't say so.
When there is a need to test more than one component/obj in a single unit test, I use those to clearly separate unit tests for different pieces.

Like
```python
class TestA:
  def test_a
  def test_b

class TestB:
  def test_a
  def test_b
```
instead of
```python
  def test_A_a
  def test_A_b
  def test_B_a
  def test_B_b
```
While I found it helpful to make unit tests visually distinct, it is just my preference.

Is there something in particular that is bothering you with this approach?

Reply:
so it is just to have separate "namespace"? do you think the tests will share some common values or so?

Reply:
In general? Yes.
Here? They can share some inputs, like query, conversation_id, ...